FluidNC/src/Spindles/BESCSpindle.cpp

```diff
@@ -58,6 +58,7 @@ namespace Spindles {
         stop();
         init_atc();
         config_message();
+        set_besc_duty(0);
     }
```

Comment:
Why not just call set_output(0) here instead of introducing the new set_besc_duty() factor?